src/App.js

```diff
@@ -8,7 +8,7 @@ const App = () => {
    <>
     <Header/>     
       <Routes>
-         <Route path="/" element={<LandingPage/>} />
+         <Route path="/" element={<LandingPage/>} exact />
```

Comment:
You can also add * path and 404 not found page

Reply:
will add it . Thank you